zerver/worker/queue_processors.py

```diff
@@ -427,6 +430,18 @@ def consume(self, event):
 class OutgoingWebhookWorker(QueueProcessingWorker):
     def consume(self, event):
         # type: (Mapping[str, Any]) -> None
-
-        # TODO: Implement this worker.
-        pass
+        message = event['message']
+        services = get_bot_services(event['user_profile_id'])
+        rest_operation = {'method': 'POST',
+                          'relative_url_path': '',
+                          'request_kwargs': {},
+                          'base_url': ''}
+        dup_event = dict()
```

Comment:
A few questions:
- What's the motivation for copying `event` here?
- What's the motivation for not using `dup_event = event.copy()`?
- Does it suffice to do a "shallow copy", or do you need to do a "deep copy"?